pennylane/_device.py

```diff
@@ -109,15 +109,15 @@ class Device(abc.ABC):
         wires (int): number of subsystems in the quantum state represented by the device.
             Default 1 if not specified.
         shots (int): number of circuit evaluations/random samples used to estimate
-            expectation values of observables. For simulator devices, a value of 0 results
-            in the exact expectation value being returned. Defaults to 0 if not specified.
+            expectation values of observables. Defaults to 1000 if not specified.
```

Comment:
@johannesjmeyer, I think `analytic` should be added to the base Device class, since plugins will also use this new nomenclature. Should be easy to copy and paste it from default_qubit/default_gaussian.

We should also add some logic to disallow analytic=False with shots < 1 here I think, thoughts @co9olguy?

Reply:
Yep

Reply:
I disagree. `analytic` makes no sense for hardware-only plugins (which are the intended principal target for pennylane), so it should only be in simulator plugins. 

Also, `shots < 1` is already disallowed generally, as it makes no sense for hardware plugins and was replaced by `analytic` for simulator plugins.

Reply:
You've changed my mind :slightly_smiling_face:

Reply:
:+1:

Reply:
Yep, if `shots < 1` is now disallowed, I think we're good without adding more logic.

I have no opinion on the usage of `analytic` though :smile: